scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/TaskLauncher.java

```diff
@@ -166,6 +175,7 @@ public void doTask(ExecutableContainer executableContainer, TaskResult[] previou
             logger.info("Output space: " + context.getNodeDataSpaceURIs().getOutputURI());
             logger.info("User space: " + context.getNodeDataSpaceURIs().getUserURI());
             logger.info("Global space: " + context.getNodeDataSpaceURIs().getGlobalURI());
+            logger.info("Scheduler url: " + context.getRestUrl());
```

Comment:
_Scheduler url:_ --> _Scheduler REST URL:_ to prevent confusion with ProActive URLs of the Scheduler?